tensorflow/compiler/mlir/xla/transforms/legalize_tf.cc

```diff
@@ -1687,6 +1687,27 @@ class ConvertSplitOp : public OpRewritePattern<TF::SplitOp> {
   }
 };
 
+class ConvertUniqueOp : public OpRewritePattern<TF::UniqueOp> {
+ public:
+  using OpRewritePattern::OpRewritePattern;
+
+  PatternMatchResult matchAndRewrite(TF::UniqueOp op,
+                                     PatternRewriter &rewriter) const override {
+    Location loc = op.getLoc();
+   
+    SmallVector<Value, 4> slices;
```

Comment:
Why 4 if you onle add 2 values to it? Also could an initializer list below work instead?